packages/react-components/react-virtualizer/package.json

```
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

_nit:_ Can you move this above `@fluentui/react-utilities` in line `34` to keep alphabetical order?

```suggestion
    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
    "@fluentui/react-utilities": "^9.8.0",
```